autopilot/migrator/alerts.go

```diff
@@ -17,11 +18,27 @@ var (
 	alertOngoingMigrationsID = alerts.RandomAlertID() // constant until restarted
 )
 
+func filterMigrationFailedAlertIDs(a []alerts.Alert) (ids []types.Hash256) {
+	for _, alert := range a {
+		var sk object.EncryptionKey
+		if val, ok := alert.Data["slabKey"]; !ok {
+			continue
+		} else if sks, ok := val.(string); !ok {
+			continue
+		} else if err := sk.UnmarshalText([]byte(sks)); err != nil {
+			continue
+		} else if alert.ID == alerts.IDForSlab(alertMigrationID, sk) {
+			ids = append(ids, alert.ID)
+		}
+	}
+	return
+}
+
 func newMigrationFailedAlert(slabKey object.EncryptionKey, health float64, objects []api.ObjectMetadata, err error) alerts.Alert {
 	data := map[string]interface{}{
 		"error":   err.Error(),
 		"health":  health,
-		"slabKey": slabKey.String(),
+		"slabKey": slabKey.String(), // used to clear migration alerts
```

Comment:
This isn't an addition, but should this even be exposed?

Reply:
I would probably not expose it again, but there's no reason we can't expose it right?